src/workerd/api/blob.h

```diff
@@ -121,7 +120,8 @@ class File: public Blob {
   // lock. This is currently only the case when parsing FormData outside of running
   // JavaScript (such as in the internal fiddle service).
   File(kj::Array<byte> data, kj::String name, kj::String type, double lastModified);
```

Comment:
Why not change this constructor as well to consolidate the `else` clause here: https://github.com/cloudflare/workerd/pull/3830/files#diff-d1203721eeacfc2a660056488a1e47d12e63017b164f6b1c7b01290c47dd3678R185 ?

Reply:
We can't really. In this case we don't have the isolate lock so we end up having to keep the data as a kj::Array rather than an v8::ArrayBuffer/v8::BackingStore. There's a case where we end up parsing FormData outside of the isolate lock and end up creating a `File` object even tho it is never actually passed off to JS. This would require an additional refactor that I'd prefer to keep separate.